src/main/java/com/minecolonies/util/BlockPosUtil.java

```diff
@@ -23,8 +23,16 @@
  */
 public final class BlockPosUtil
 {
+    /**
+     * Min distance to availate two positions as close.
+     */
     private static final double CLOSE_DISTANCE = 4.84;
 
+    /**
+     * Max distance two points should have divided by 3 because of 3dim positions.
+     */
+    private static final long MAX_SQRT = 3_000_000_000L / 3;
```

Comment:
Worlds can be much bigger than this now, I thing its 60_000_000 x 60_000_000 now